sencha-workspace/SlateTasksStudent/app/view/TaskTree.js

```diff
@@ -216,16 +216,33 @@ Ext.define('SlateTasksStudent.view.TaskTree', {
         }
     },
 
+    afterTasksLoad: function() {
+        var tree = this,
+            sublists = tree.el.select('.slate-tasktree-sublist');
+
+        sublists.each(function(sublist) {
+            sublist.set({
+                'data-natural-height': sublist.getHeight(),
+                style: {
```

Comment:
@ryon you should use the `.setStyle()` api for this or the convenient wrapper `.setHeight()`, I'm concerned this form may be liable to overwrite other style attributes.